pyne/dbgen/materials_library.py

```diff
@@ -128,8 +128,7 @@ def parse_materials(mats, lines):
 # Writes to file
 def make_materials_compendium(nuc_data, matslib):
     """Adds materials compendium to nuc_data.h5."""
-    matslib.write_hdf5(nuc_data, datapath="/material_library/materials",
-                       nucpath="/material_library/nucid")
+    matslib.write_hdf5(nuc_data, datapath="/materials")
```

Comment:
Given that `/materials` is the default path in the C++ API for this object, we shouldn't need to provide it here any longer. True here and in a few other locations.

Reply:
I'm noting now that this is only true for `write_hdf5`. A separate issue, but it seems like it would make sense for `from_hdf5` to have the same default value for the datapath.

Reply:
this is the name of the material library that will contain the written library.
it will be nested inside the `material_library` dataset

Reply:
Agreed, based on your comment above, but we're still explicitly providing the default value of the `datapath` argument here. 

https://github.com/pyne/pyne/blob/c61feaca4748299b08aeec7f069c221684da8e45/pyne/material_library.pyx#L122

It seems like this could be shortened this to:

```python
matslib.write_hdf5(nuc_data)
```